packages/core/package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "@datadog/build-plugins-core",
+    "packageManager": "yarn@4.0.2",
+    "license": "MIT",
+    "private": true,
+    "author": "Datadog",
+    "main": "./src/index.js",
+    "exports": {
+        "./*": {
+            "import": "./src/*.ts",
+            "require": "./src/*.ts"
+        },
+        "./hooks/datadog": {
```

Comment:
2 things:
1. I think this needs to go before the `*`
2. if you're using the same for `import` & `require`, you can remove both, and just do `"./hooks/datadog": "./src/hooks/datadog/index.ts"` (same for `./*`)